src/Model/GroundWaterTransport/gwt1cnc1.f90

```diff
@@ -486,4 +487,33 @@ subroutine cnc_rp_ts(this)
     return
   end subroutine cnc_rp_ts
 
+  !> @ brief Return a bound value
+  !!
+  !!  Return a bound value associated with an ncolbnd index and row.
+  !<
+  function cnc_bound_value(this, col, row) result(bndval)
+    ! -- modules
+    use ConstantsModule, only: DZERO
+    ! -- dummy variables
+    class(GwtCncType), intent(inout) :: this !< BndExtType object
+    integer(I4B), intent(in) :: col
+    integer(I4B), intent(in) :: row
+    ! -- result
+    real(DP) :: bndval
+    !
+    select case (col)
+    case (1)
+      bndval = this%tspvar(row)
```

Comment:
This should actually be reporting the scaled quantity when iauxmultcol is greater than 0.  The chd package needs to be updated as well (it was the first and it was an oversight), you could considering that update in this PR.  See ghb/drn/riv for comparison as those should all report the scaled quantity.

Reply:
Resolved with 2babaa8.  Need to see how the CI turns out.

Reply:
CI looks to be passing